app/models/solr_document.rb

```diff
@@ -73,7 +73,7 @@ def linked_publications(params)
     response, _deprecated_stuff = search_service.search_results
     docs = response.documents
     tuples = docs.map do |doc|
-      "<li><a href=\"#{search_link(doc.id)}\">#{doc.title}</a></li>"
+      "<li><a href=\"#{search_link(doc.id)}\">#{doc.title}</a> (#{doc['created_year_isim'].first})</li>"
```

Comment:
how about `doc.first('created_year_isim')` instead.  Does that work?

Reply:
Works for me.